crossbundle/tools/src/commands/common/gen_minimal_project/consts.rs

```diff
@@ -1,27 +1,37 @@
 pub const BEVY_CARGO_TOML_VALUE: &str = r#"
 [package]
 name = "example"
+version = "0.1.3"
```

Comment:
Version in examples shouldn't follow the crossbow version. Replace with `0.1.0`.